Source/WebCore/platform/graphics/gstreamer/mse/MediaPlayerPrivateGStreamerMSE.cpp

```diff
@@ -405,7 +421,8 @@ void MediaPlayerPrivateGStreamerMSE::sourceSetup(GstElement* sourceElement)
 
     if (m_mediaSourcePrivate && m_mediaSourcePrivate->hasAllTracks()) {
         m_tracks = filterOutRepeatingTracks(m_tracks);
-        webKitMediaSrcEmitStreams(WEBKIT_MEDIA_SRC(m_source.get()), m_tracks);
+        auto playbackTracks = filterOutTextTracks(m_tracks);
+        webKitMediaSrcEmitStreams(WEBKIT_MEDIA_SRC(m_source.get()), playbackTracks);
```

Comment:
Seems like `webKitMediaSrcEmitStreams()` can now take a `Vector<>&&` (each track could be moved to the `Stream`)? If so, WTFMove can be used here.

Reply:
`webKitMediaSrcEmitStreams()` would need to be updated as well.

Reply:
I agree with Alicia now, so I changed `webKitMediaSrcEmitStreams` back.